plugins/fabrik_form/php/php.php

```diff
@@ -203,7 +203,9 @@ public function onBeforeCalculations()
 		{
 			if ($this->_runPHP() === false)
 			{
-				return JError::raiseWarning(E_WARNING, 'php form plugin failed');
+				$code = $params->get('curl_code', '');
+				$error = error_get_last();
+				return JError::raiseWarning(E_WARNING, 'php form plugin failed:<br>'.$code.'<br>'.$error['type']. $error['message']);
```

Comment:
I don't think appending the code to the warning is a good idea, as this gets shown to the user.  Showing the error message is OK.  Or perhaps check to see if debug mode is enabled, and only append the code to the msg if it is.  You can use FabrikHelper::isDebug() to determine if debug mode is in effect.